docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -17,6 +17,81 @@
 
 Converts `value` to [Decimal](../../data_types/decimal.md) of precision `S`. The `value` can be a number or a string. The `S` (scale) parameter specifies the number of decimal places.
 
+## toDecimal*OrNull
+
+Converts an input string to the value of [Nullable(Decimal(P,S))](../../data_types/decimal.md) data type. This family of functions include:
+
+- `toDecimal32OrNull( expr, S)` — Results with Nullable(Decimal32(S)) data type.
+- `toDecimal64OrNull( expr, S)` — Results with Nullable(Decimal64(S)) data type.
+- `toDecimal128OrNull( expr, S)` — Results with Nullable(Decimal128(S)) data type.
+
+**Parameters**
+
+- `expr` — Any of ClickHouse [expressions](../syntax.md#syntax-expressions), returning a value of the [String](../../data_types/string.md) data type. For example, `'1.111'`.
+- `S` — Number of decimal places in the resulting value.
```

Comment:
S - scale. Decimal type contains 2 args: precision and scale. We set precision indirectly by using maximum precisions that could be set for Decimal32, Decimal64 and Decimal128. But we cannot avoid setting scale here. Scale means how much digits (not 'place') the number could have after dot.